README.md

```diff
@@ -1,34 +1,41 @@
 [![Build Status](https://ci.eclipse.org/codewind/buildStatus/icon?job=Codewind%2Fcodewind-eclipse%2Fmaster)](https://ci.eclipse.org/codewind/job/Codewind/job/codewind-eclipse/job/master/)
 [![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
 
+# Codewind for Eclipse
+Create and develop cloud-native, containerized web applications from Eclipse.
+
 ## Installing Codewind for Eclipse
+You can install Codewind locally in Eclipse. For more information about installing Codewind, see [Getting started: Codewind for Eclipse](https://www.eclipse.org/codewind/mdteclipsegettingstarted.html).
+
 Prerequisites
 - Download and install the latest [Eclipse IDE for Java EE Developers](https://www.eclipse.org/downloads/packages/release/) or use an existing installation. The earliest supported version of the Eclipse IDE for Codewind for Eclipse is 4.11.0 (2019-03).
 - Install Docker.
 - If you use Linux, you also need to install Docker Compose.
 
 Complete the installation:
 1. Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
-2. Open the **Codewind Explorer** view.
+2. Open the **Codewind Explorer** view. Go to **Window**>**Show View**>**Other…**>**Codewind**>**Codewind Explorer**.
 3. Double-click the **Codewind** entry in the view to finish installing Codewind. The download is approximately 1 GB. For more information, see [Installing Codewind for Eclipse](https://www.eclipse.org/codewind/mdteclipseinstallinfo.html).
 
 ## Using Codewind for Eclipse
 Right-click the **Local Projects** entry in the view to create new projects or add existing projects to Codewind. After a project is created or added, it displays in the **Codewind Explorer** view. Right-click the project to see the available actions.
 
-Features:</br>
-- Create new projects from application templates or add existing container-ready projects to Codewind.
-- View Codewind projects, including application and build statuses.
-- Debug **Microprofile/Java EE** and **Spring** projects in their containers.
-- Set up a Chrome debug session for **Node.js** projects.
-- View application and build logs in the **Console** view.
-- View and edit project deployment information.
-- Open a shell session into a Codewind application container.
-- Toggle project auto build and manually initiate project builds.
-- Integrate Codewind validation errors into the **Markers** view.
-- Disable, enable, and remove projects.
+[Features:](https://www.eclipse.org/codewind/mdteclipsemanagingprojects.html)</br>
+- **Open Application**: Opens the application in the default Eclipse browser. This action is only available when the application is running or debugging.
+- **Open Project Overview**: Opens the overview page for a project. You can use this action to see information about the project, enable or disable the project, turn auto build on and off, or edit project settings.
```

Comment:
You can no longer enable/disable the project or turn auto build on/off from the overview page in Eclipse.  You can still edit the project settings.